src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -168,6 +168,51 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="PackageReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="PackageReference" _locComment="" -->Reference to a package
```

Comment:
I don't what the process for localization/insertion is but from the setup side it's picked up on the Community resource package from under the "Localized" sub folders in the binary. Layout wise it's dropped under <VsInsallDir>\Xml\Schemas\MsBuild